packages/language-server/src/plugins/typescript/astro2tsx.ts

```diff
@@ -3,12 +3,28 @@ import { parseAstro } from '../../core/documents/parseAstro';
 
 function addProps(content: string, className: string): string {
 	let defaultExportType = 'Record<string, any>';
+	let shouldAddGlobal = false;
+	let astroGlobal = "type AstroGlobal = import('astro').AstroGlobal";
```

Comment:
This will work even in cases where Astro (the package, not Atlanta) is unavailable because `import('astro')` is then `any`